docs/security.html

```diff
@@ -261,6 +261,139 @@
 </ol>
 
 <h3><a id="security_authz">7.4 Authorization and ACLs</a></h3>
+Kafka ships with a pluggable Authorizer and an out-of-box authorizer implementation that uses zookeeper to store all the acls. Kafka acls are defined in the general format of "Principal P is [Allowed/Denied] Operation O From Host H On Resource R". You can read more about the acl structure on KIP-11. In order to add, remove or list acls you can use the Kafka authorizer CLI.
```

Comment:
I prefer not to mix user-facing and design docs. 

If there is information in KIP-11 that we believe users will need, we should add it here.